widget.json

```
@@ -73,10 +73,10 @@
       "vendor/mixitup-multifilter.min.js",
       "vendor/autosize.js",
       "vendor/unorm.js",
+      "vendor/ellipssis.js",
```

Change the file name to match how the library was intended.

```suggestion
      "vendor/ellipsed.js",
```